package.json

```diff
@@ -16,26 +16,27 @@
     "LICENSE"
   ],
   "scripts": {
-    "test": "jshint lib index.js && jscs lib index.js && mocha -t 5000 -b -R spec test/index",
-    "cover": "nyc --reporter=lcov --reporter=text-summary npm test"
+    "lint": "eslint .",
+    "test": "jshint lib index.js && mocha -t 5000 -b -R spec test/index",
+    "cover": "eslint . && nyc --reporter=lcov --reporter=text-summary npm test"
   },
   "dependencies": {
     "extend": "^3.0.0",
-    "findup-sync": "^2.0.0",
+    "findup-sync": "^3.0.0",
     "fined": "^1.0.1",
     "flagged-respawn": "^1.0.0",
     "is-plain-object": "^2.0.4",
     "object.map": "^1.0.0",
-    "rechoir": "^0.6.2",
+    "rechoir": "^0.7.0",
```

Comment:
I think something breaking happened in that bump.

Reply:
Breaking is by [rechoir#37](https://github.com/gulpjs/rechoir/pull/37) ? Is it better to defer this bump until liftoff's major bump?

Reply:
Yeah, I think we should defer that.  I'm still nervous we didn't catch all the edge cases with that feature.